httpx/_config.py

```diff
@@ -251,17 +288,11 @@ def __init__(
                 )
                 timeout = None
             self.connect_timeout = (
```

Comment:
Note that I haven't changed the `Timeout.<xyz>_timeout` attributes yet. I _could_, but I don't actually think they're public API, are they? If not, then I'd like to leave that renaming for a follow-up cleanup PR.

Reply:
I think we'd *potentially?* be okay without a deprecation on those, but happy either way.

(Given that they're very unlikely to be used, aren't documented, and that we have a clear policy on 0.x version changes / pinning expectations and nice, clear versioning notes)

Reply:
That's my feeling as well… :-) If so I can make the change in this PR, since it's contained to the `Timeout` class (transports expect a timeout `dict` and those keys don't change), and maybe some tests.